akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -51,6 +51,7 @@ private[persistence] trait Eventsourced extends Snapshotter with Stash with Stas
   private val writerUuid = UUID.randomUUID.toString
 
   private var journalBatch = Vector.empty[PersistentEnvelope]
+  // no longer used, but kept for binary compatibility
```

Comment:
there is also `def maxBatchSizeReached` perhaps be removed, or at least not called any more